pkg/process/procutil/process_linux.go

```diff
@@ -364,6 +366,20 @@ func (p *probe) getCmdline(pidPath string) []string {
 	return trimAndSplitBytes(cmdline)
 }
 
+// getCommandName retrieves the command name from "comm" file for a process in procfs
+func (p *probe) getCommandName(pidPath string) string {
```

Comment:
Would you mind quickly adding a test for this? Just want to get our code coverage up 😄

Reply:
I added a basic test with a hardcoded check against one of the fake procfs entries. I think it serves its purpose, but let me know if you had something else in mind.

Note that most tests for this class's methods test the output against the [gopsutil library](https://github.com/DataDog/gopsutil), but `comm` is not collected by gopsutil.